peripherals/dma/dma_v1.yaml

```diff
@@ -0,0 +1,74 @@
+# DMA as used on F0, F1, F3 family
+#
+# Please use `dma_v1_with_remapping.yaml` for devices that has DMA remapping
+# functionality such as the L0 and L4 family.
+
+# DMA1 has 7 channels
+# DMA2 has 5 channels
+"DMA[12]":
+  ISR:
+    "TEIF*":
+      NoError: [0, "No transfer error"]
+      Error: [1, "A transfer error has occured"]
+    "HTIF*":
+      NotHalt: [0, "No half transfer event"]
+      Half: [1, "A half transfer event has occured"]
+    "TCIF*":
+      NotComplete: [0, "No transfer complete event"]
+      Complete: [1, "A transfer complete event has occured"]
+    "GIF*":
+      NoEvent: [0, "No transfer error, half event, complete event"]
+      Event: [1, "A transfer error, half event or complete event has occured"]
+  IFCR:
+    "CTEIF*":
+      NoEffect: [0, "No effect"]
+      Clear: [1, "Clears the TEIF flag in the ISR register"]
+    "CHTIF*":
+      NoEffect: [0, "No effect"]
+      Clear: [1, "Clears the HTIF flag in the ISR register"]
+    "CTCIF*":
+      NoEffect: [0, "No effect"]
+      Clear: [1, "Clears the TCIF flag in the ISR register"]
+    "CGIF*":
+      NoEffect: [0, "No effect"]
+      Clear: [1, "Clears the GIF, TEIF, HTIF, TCIF flags in the ISR register"]
```

Comment:
Is the `NoEffect` necessary?

Reply:
These are write-only registers and the reset value is 0x0 so I wouldn't have thought so. The only use case I can find is if your code is always writing some value to the field (e.g. writing an argument that is supplied) and needs to either write 0 or 1. However in many other places we've just got the `Clear` without a corresponding `NoEffect` and it's not caused problems yet, so I think leave it out.